compiler/damlc/daml-opts/daml-opts-types/DA/Daml/Options/Types.hs

```diff
@@ -86,8 +86,8 @@ data DlintUsage
   deriving Show
 
 data ScenarioValidation
-    = ScenarioValidationLight
-    | ScenarioValidationFull
+    = ScenarioValidationDisable
```

Comment:
If we only have disable/enable states, I would just make this a newtype around a `Bool`.